tests/db_tests/test_hybrid_properties.py

```diff
@@ -54,12 +68,16 @@ def test_itl_regions_does_not_raise_uncaught_exception_with_invalid_postcode():
         project_id="1",
         programme_junction_id="1",
         project_name="Project 1",
-        primary_intervention_theme="Theme 1",
-        location_multiplicity="MULTIPLE",
+        event_data_blob=json.dumps(
+            {
+                "primary_intervention_theme": "Theme 1",
+                "location_multiplicity": "MULTIPLE",
+                "locations": "Example Location",
+                "gis_provided": "Yes",
+                "lat_long": "12345",
+            }
+        ),
         postcodes=["ZZ1 2AB"],  # null postcodes
```

Comment:
not really part of this PR, but I noticed this comment is wrong. Could you update to say "invalid postcode"